releasenotes/notes/0.11/prepare-0.11-af688e532712c830.yaml

```diff
@@ -0,0 +1,66 @@
+---
+prelude: |
+  This release includes many new features and bug fixes. The highlights of this
+  release are more methods to improve working with edges in both the
+  :class:`~retworkx.PyGraph` and :class:`~retworkx.PyDiGraph` classes,
+  several new graph generators, and a new set of interactive traversal
+  functions :func:`~retworkx.bfs_search`, :func:`~retworkx.dfs_search`,
+  :func:`~retworkx.dijkstra_search`, and :class:`~retworkx.TopologicalSorter`
+  which enable iteratively interacting with the graph during different types
+  of traversal.
+
+  This release also introduces a new separate Rust crate ``retworkx-core`` which
+  is a library for use in Rust that's built on top of
+  `petgraph <https://github.com/petgraph/petgraph>`__ that extends the
+  functionality provided by petgraph. The functionality in this crate is generic
+  and can work with any petgraph graph, not just the :class:`~retworkx.PyGraph`
+  amd :class:`~retworkx.PyDiGraph`.
+
+  The 0.11.0 release fully supports Python 3.10 and precompiled binaries
+  for Python 3.10 are published on PyPI (previous releases worked with 3.10 but
+  required compiling from source to install). This is also the last release to
+  support Python 3.6, starting in retworkx 0.12.0 Python >=3.7 will be required
+  to run retworkx. Additionally, for users compiling retworkx from source this
+  will be the last release with a minimum supported Rust version (MSRV) of 1.41,
+  in retworkx 0.12.0 the MSRV will be increased to 1.48.
+
+features:
+  - |
+    Added a new workspace crate, ``retworkx-core`` as part of retworkx. This is
```

Comment:
Similar question about linking to crates.io here.

Reply:
Done in: https://github.com/Qiskit/retworkx/pull/549/commits/b16f1549d0297a7459cd5332e19b97e316d7b99f